### PR TITLE
[Teams] Add chat rsc to permissions reference

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| TeamsTab.Read.Chat             | Read this chat's tabs.                                        | Allows the app to read this chat's tabs, without a signed-in user. |No | No |
+| TeamsTab.Create.Chat           | Create tabs in this chat.                                     | Allows the app to  create tabs in this chat, without a signed-in user. |No | No |
+| TeamsTab.Delete.Chat           | Delete this chat's tabs.                                      | Allows the app to delete this chat's tabs, without a signed-in user. |No | No |
+| TeamsTab.ReadWrite.Chat        | Manage this chat's tabs.                                      | Allows the app to manage this chat's tabs, without a signed-in user. |No | No |
+| TeamsAppInstallation.Read.Chat | Read which apps are installed in this chat.                   | Allows the app read the Teams apps that are installed in this chat along with the permissions granted to each app, without a signed-in user.  |No | No |
```
Comment (AkJo) [inline concepts/permissions-reference.md:538]: app to read

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| OnlineMeeting.ReadBasic.Chat   | Read basic properties of a meeting associated with this chat. | Allows the app to read basic properties of a meeting associated with this chat, without a signed-in user. |No | No |
```
Comment (AkJo) [inline concepts/permissions-reference.md:539]: Allows the app to read basic properties, such as name, schedule, organizer and join link, of a meeting associated with this chat, without a signed-in user.

File: concepts/permissions-reference.md [hunk 3]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 > **Note:** For messages in a channel, see [ChannelMessage permissions](#channel-message-permissions).
 
+## Chats resource-specific consent permissions
```
Comment (AkJo) [inline concepts/permissions-reference.md:525]: Thinking of keeping this singular.

Chat resource specific consent permissions

Comment (AkJo) [reply concepts/permissions-reference.md:525]: What are your thoughts?

Comment (jessiechan27) [reply concepts/permissions-reference.md]: I also think it should be singular to keep consistent with the other resources

**Review by AkJo — APPROVED** [review]

File: concepts/permissions-reference.md [hunk 4]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
 
+## Chat resource-specific consent permissions
+
+| Permission                     | Display String                                                | Description  | Admin Consent Required | Microsoft Account supported |
+|:-------------------------------|:--------------------------------------------------------------|:-------------|:-----------------------|:----------------------------|
+| ChatSettings.Read.Chat         | Read this chat's settings.                                    | Allows the app to read this chat's settings, without a signed-in user. |No | No |
+| ChatSettings.ReadWrite.Chat    | Read and write this chat's settings.                          | Allows the app to read and write this chat's settings, without a signed-in user. |No | No |
+| ChatMessage.Read.Chat          | Read this chat's messages.                                    | Allows the app to read this chat's messages, without a signed-in user. |No | No |
+| ChatMember.Read.Chat           | Read this chat's members.                                     | Allows the app to read the members of this chat, without a signed-in user. |No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:532]: For consistency, I recommend you replacing "the members of this chat" with "this chat's members"

File: concepts/permissions-reference.md [hunk 5]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 > **Note:** For messages in a channel, see [ChannelMessage permissions](#channel-message-permissions).
 
+## Chat resource-specific consent permissions
+
+| Permission                     | Display String                                                | Description  | Admin Consent Required | Microsoft Account supported |
+|:-------------------------------|:--------------------------------------------------------------|:-------------|:-----------------------|:----------------------------|
+| ChatSettings.Read.Chat         | Read this chat's settings.                                    | Allows the app to read this chat's settings, without a signed-in user. |No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:529]: For consistency, I recommend you using italics to emphasize all permissions listed in this table. For example, \_ChatSettings.ReadWrite.Chat\_

File: concepts/permissions-reference.md [hunk 6]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| TeamsTab.Create.Chat           | Create tabs in this chat.                                     | Allows the app to create tabs in this chat, without a signed-in user. |No | No |
+| TeamsTab.Delete.Chat           | Delete this chat's tabs.                                      | Allows the app to delete this chat's tabs, without a signed-in user. |No | No |
+| TeamsTab.ReadWrite.Chat        | Manage this chat's tabs.                                      | Allows the app to manage this chat's tabs, without a signed-in user. |No | No |
+| TeamsAppInstallation.Read.Chat | Read which apps are installed in this chat.                   | Allows the app to read the Teams apps that are installed in this chat along with the permissions granted to each app, without a signed-in user.  |No | No |
+| OnlineMeeting.ReadBasic.Chat   | Read basic properties of a meeting associated with this chat. | Allows the app to read basic properties, such as name, schedule, organizer and join link, of a meeting associated with this chat, without a signed-in user. |No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:539]: For readability, I recommend you using em dash after "... basic properties". Suggested revision:

"Allows the app to read basic properties—such as name, schedule, organizer, and join link—of a meeting associated with this chat, without a signed-in user."

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: concepts/permissions-reference.md [hunk 7]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _TeamsTab.Create.Chat_           | Create tabs in this chat.                                     | Allows the app to create tabs in this chat, without a signed-in user. |No | No |
+| _TeamsTab.Delete.Chat_           | Delete this chat's tabs.                                      | Allows the app to delete this chat's tabs, without a signed-in user. |No | No |
+| _TeamsTab.ReadWrite.Chat_        | Manage this chat's tabs.                                      | Allows the app to manage this chat's tabs, without a signed-in user. |No | No |
+| _TeamsAppInstallation.Read.Chat_ | Read which apps are installed in this chat.                   | Allows the app to read the Teams apps that are installed in this chat along with the permissions granted to each app, without a signed-in user.  |No | No |
+| _OnlineMeeting.ReadBasic.Chat_   | Read basic properties of a meeting associated with this chat. | Allows the app to read basic properties —such as name, schedule, organizer and join link— of a meeting associated with this chat, without a signed-in user. |No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:539]: Please, remove the empty character before and after the em dashes. For more information, see https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#dont-be-spacey. Additionally, add a comma after organizer. Suggested revision:

Allows the app to read basic properties—such as name, schedule, organizer, and join link—of a meeting associated with this chat, without a signed-in user.

File: concepts/permissions-reference.md [hunk 8]
```diff
@@ -522,6 +522,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+|:-------------------------------|:--------------------------------------------------------------|:-------------|:-----------------------|:----------------------------|
+| _ChatSettings.Read.Chat_         | Read this chat's settings.                                    | Allows the app to read this chat's settings, without a signed-in user. |No | No |
+| _ChatSettings.ReadWrite.Chat_    | Read and write this chat's settings.                          | Allows the app to read and write this chat's settings, without a signed-in user. |No | No |
+| _ChatMessage.Read.Chat_          | Read this chat's messages.                                    | Allows the app to read this chat's messages, without a signed-in user. |No | No |
+| _ChatMember.Read.Chat_           | Read this chat's members.                                     | Allows the app to read this chat's members, without a signed-in user. |No | No |
+| _Chat.Manage.Chat_               | Manage this chat.                                             | Allows the app to manage the chat, the chat's members and grant access to the chat's data, without a signed-in user.  |No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:533]: Please, add a comma after chat's members. Suggested revision:

"Allows the app to manage the chat, the chat's members, and grant access to the chat's data, without a signed-in user."

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**